src/site/xdoc/server/config-cassandra.xml

```diff
@@ -76,22 +76,27 @@
         # cassandra.pooling.timeout=5000
         ## In seconds.
         # cassandra.pooling.heartbeat.timeout=30
+        # cassandra.pooling.max.queue.size=256
 
         <dt><strong>cassandra.pooling.local.max.connections</strong></dt>
         <dd>Optional.
-          If specified defines the Cassandra <a href="http://docs.datastax.com/en/developer/java-driver/2.1/manual/pooling/">maximum number of connections to hosts</a> (remote and local)</dd>
+          If specified defines the Cassandra <a href="https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/">maximum number of connections to hosts</a> (remote and local).</dd>
```

Comment:
will more precise when adding `#pool-size` at the end of the url (https://docs.datastax.com/en/developer/java-driver/3.5/manual/pooling/#pool-size) 👍